app/services/foreman_ansible/playbooks_importer.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Imports playbooks from smart proxy
+  class PlaybooksImporter
+    include ::ForemanAnsible::ProxyAPI
+    delegate :playbooks, :playbooks_names, :to => :proxy_api
+
+    def initialize(proxy = nil)
+      @ansible_proxy = proxy
+    end
+
+    def import_playbooks(playbooks_names)
+      playbooks = playbooks(playbooks_names)
+      result = { created: {}, updated: {} }
+      playbooks.each do |playbook|
+        parsed_playbook = parse_playbook playbook
+        job_template_id = JobTemplate.where(name: parsed_playbook[:name]).pick(:id)
+        if job_template_id.present?
+          updated = update_job_template(job_template_id, parsed_playbook)
+          result[:updated].merge!(updated) unless updated.nil?
+        else
+          result[:created].merge!(create_job_template(parsed_playbook))
+        end
+      end
+      result
+    end
+
+    def parse_playbook(playbook)
+      content = playbook['playbooks_content']
+      {
+        name: playbook['name'],
+        playbook_content: add_metadata(playbook['name']) + content,
+        vars: get_vars(content)
+      }
+    end
+
+    def add_metadata(playbook_name)
+      <<END_HEREDOC
```

Comment:
The best here would be 
```suggestion
      <<~END_HEREDOC
```
this version allows the closing to be indented, but the result is not indented, which is what we want here IMHO ?